app/src/Controller/AdviceController.php

```diff
@@ -7,6 +7,7 @@
 use Symfony\Component\HttpFoundation\Response;
 use App\Manager\GameManager;
 use App\Manager\AdvisorManager;
+use App\Domain\Coords;
```

Comment:
I'm not a big fan of this `Domain` name ... What is the domain? `QuixoGame` maybe ?

Reply:
Yes I could change that !